index.html

```diff
@@ -35,11 +35,11 @@
         <div class="body-class">
                 
```

Comment:
### Image 1
![cards (copy)](https://user-images.githubusercontent.com/93039359/178313682-6e4e7b4b-bdce-4bf4-b805-173a970fb568.png)

### Image 2
![color-blind (copy)](https://user-images.githubusercontent.com/93039359/178313716-44cb69d9-4be9-4520-b258-d9e625ab8cca.png)

### Image 3
![forms-of-focus (copy)](https://user-images.githubusercontent.com/93039359/178313736-409ff56c-f2b8-4db7-996b-1777888ea9f6.png)

**Changes**
- [ ] You don't have a color contrast of 4.5:1 for these texts shown in the images. 
- [ ] Please add CSS :focus effect on each input item to make it visible for color-blind people focusing.
- [ ] Please add labels to each input item for accessibility purposes.
- [ ] Please make the placeholders a little bigger. Please fix them :+1:

Reply:
- [ ] Kindly make your placeholder a little bigger and bolder as it is currently faint. This enables those with eye impairment to see clearly what is being written. Please ensure you adjust it as noted by the previous reviewer
- 
![image](https://user-images.githubusercontent.com/63473584/178366714-eb21ca1e-cbf4-453a-a88c-d1827005efa3.png)